modules/rostests/rosautotest/CConfiguration.cpp

```diff
@@ -75,6 +78,12 @@ CConfiguration::ParseParameters(int argc, wchar_t* argv[])
                     m_Submit = true;
                     break;
 
+                case 't':
+                    ++i;
+                    tmp_RepeatCount = _wtoi(argv[i]);
```

Comment:
Not sure I see the point in using the `tmp_` variable. Since `m_RepeatCount` is going to be overwritten unconditionally in this case and we're not examining the previous value, might as well write there directly?

Reply:
The idea here was to truncate input. I don't want the whole `int` range

Reply:
I'd argue it's better to fail than to silently truncate. But if you feel strongly, the temp var is fine.